app.py

```diff
@@ -34,6 +36,29 @@ def azul_to_dos(azul):
     return data_object
 
 
+def azul_to_bdl(result):
+    """
+    Takes an Azul ElasticSearch result and converts it to a DOS data
+    bundle.
+
+    :param result: the ElasticSearch result dictionary
+    :return: DataBundle
+    """
+    azul = result['_source']
+    bundle = {
+        'id': azul['id'],
+        'version': azul['version'],
+        'checksums': [{'checksum': c.split(':')[0], 'type': c.split(':')[1]} for c in azul['checksums']],
```

Comment:
To really show this, we'll want a list of resolvable data object identifiers in here.